samples/unit-tests/series-bubble/extremes/demo.js

```diff
@@ -91,3 +91,36 @@ QUnit.test('Setting X axis extremes on bubble (#5167)', function (assert) {
     assert.ok(chart.yAxis[0].min < 102.9, 'DE point is within range');
     assert.ok(chart.yAxis[0].max > 102.9, 'DE point is within range');
 });
+
+QUnit.test('Calculation of Z extremes when Z = 0 (#17280)', function (assert) {
+    const chart = Highcharts.chart('container', {
```

Comment:
Thanks for the test! Is there a chance to use an already existing test instead of creating a whole new separate test?
Word of explanation: a separate new test creates the whole chart from scratch. When having thousands of tests, we would have to create thousands of charts, and that would be a performance kill (running tests already takes at least a few minutes). Instead, you can use the existing test/chart and use the `update()` method (very often without `redraw` if not needed) - it's much more efficient for a longer period of time.